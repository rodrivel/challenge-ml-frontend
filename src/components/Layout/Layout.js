import React from 'react';
import PropTypes from 'prop-types';
import { Container, Col, Row } from 'react-grid-system';
import cx from './Layout.module.scss';

function Layout(props) {
  return (
    <>
      <Container className={cx.Layout__Container}>
        <Row className={cx.Layout__Container__Row}>
          <Col xs={12} xxl={10} offset={{ xxl: 1 }}>
            { props.children }
          </Col>
        </Row>
      </Container>
    </>
  );
}

Layout.propTypes = {
  children: PropTypes.oneOfType([
    PropTypes.object,
    PropTypes.array
  ])
}

export default Layout;