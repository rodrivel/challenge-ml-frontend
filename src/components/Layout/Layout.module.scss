.Layout__Container {
  width: 100%;

  .Layout__Container__Row {
    justify-content: center !important;
  }
}