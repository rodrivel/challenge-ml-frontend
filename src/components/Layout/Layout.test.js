import React from 'react';
import { shallow } from 'enzyme';
import Layout from './Layout';

it('expect to render Layout component', () => {
  expect(shallow(<Layout />)).toMatchSnapshot();
});
