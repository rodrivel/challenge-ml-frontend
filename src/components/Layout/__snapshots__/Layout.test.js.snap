// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Layout component 1`] = `ShallowWrapper {}`;
